src/_pytest/assertion/util.py

```diff
@@ -123,13 +124,21 @@ def isiterable(obj):
 
 def assertrepr_compare(config, op, left, right):
     """Return specialised explanations for some operators/operands"""
-    maxsize = (80 - 15 - len(op) - 2) // 2  # 15 chars indentation, 1 space around op
-    left_repr = saferepr(left, maxsize=maxsize)
-    right_repr = saferepr(right, maxsize=maxsize)
+    verbose = config.getoption("verbose")
+    if verbose > 1:
+        left_repr = safeformat(left)
```

Comment:
So the idea here is to don't use `maxsize` with `-vv`? Seems reasonable.

Please add a CHANGELOG entry as well.